### PR TITLE
test: add helm repo install check

### PR DESCRIPTION
**Pull request by umagnus**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

feat: add helm repo install check

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: hack/verify-helm-chart-index.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+INDEX=${PKG_ROOT}/charts/index.yaml
+
+function check_url() {  
+    url=$1
+    result=$(curl -I -m 5 -s -w "%{http_code}\n" -o /dev/null $1)
+    if [ $result -ne 200 ]
+    then
+        echo "Error: $1 is fail."
+        echo "Error: wrong pkg in "${PKG_ROOT}${url#*master}
```
Comment (andyzhangx) [inline hack/verify-helm-chart-index.sh:29]: if the link is invalid, this script should return `exit 1`, otherwise this script still passes

Comment (umagnus) [reply hack/verify-helm-chart-index.sh:29]: ok, it is added.

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Pull request merged**